devito/functions/__init__.py

```diff
@@ -0,0 +1,6 @@
+from .constants import *  # noqa
+from .dimension import *  # noqa
+from .dense import * # noqa
+from .sparse import *  # noqa
+from .types import *  # noqa
```

Comment:
types misleading ad this point, I'd call it basic (just like SymPy does)

Reply:
ok